src/Surfnet/StepupMiddlewareClientBundle/Configuration/Dto/InstitutionConfigurationOptions.php

```diff
@@ -34,6 +34,14 @@ class InstitutionConfigurationOptions implements Dto
     public $showRaaContactInformation;
 
     /**
+     * @Assert\Type(type="array", message="middleware_client.dto.configuration.allowed_second_factors.must_be_array")
+     * @Assert\All(
+     *     @Assert\Type(type="string", message="middleware_client.dto.configuration.allowed_second_factors.item_must_be_string")
+     * )
+     */
+    public $allowedSecondFactors;
```

Comment:
hrms, any chance we could make this `SecondFactorType`s? Would make treating them more consistent and I think easier...